fili-core/src/main/java/com/yahoo/bard/webservice/table/Schema.java

```diff
@@ -15,6 +15,15 @@
 public interface Schema {
 
     /**
+     * Retrieve a column from schema given the name of the column.
+     *
+     * @param columnName  The column name in string
+     *
+     * @return an optional of the matching name column
+     */
+    Optional<Column> getColumn(String columnName);
```

Comment:
While I don't disagree that it seems natural, if we don't have a clear use for a method beyond tests, we should be very careful with making the public surface of classes any larger than is required. Once there, it becomes _much_ harder to change it in a backwards-compatible way, and any extra surface area is extra complexity. (even if it seems small)